app/models/trip.rb

```diff
@@ -0,0 +1,20 @@
+class Trip < ApplicationRecord
+  has_many :tickets, dependent: :nullify
+  belongs_to :bus, optional: true
+  has_many :seats, dependent: :nullify
+  belongs_to :route, optional: true
+  has_many :boardings, dependent: :nullify
+  has_many :boarding_points, through: :boardings
+
+  validates :ticket_price, :total_booked, :date, :time, presence: true
+  validates :ticket_price, numericality: { greater_than: 0, less_than_or_equal_to: 10_000 }
+  validates :total_booked, numericality: { greater_than: 0 }
```

Comment:
why greater than 0? what will be the initial value of total_booked?

Reply:
Thanks for pointing it out bhaia. Chnaged it to greater_than_or_equal_to.